package.json

```diff
@@ -22,15 +22,15 @@
         "url": "git+https://github.com/trailheadapps/ebikes-lwc"
     },
     "engines": {
-        "node": ">= 8.9.4",
-        "npm": ">= 5.7.1"
+        "node": ">= 10.13.0",
+        "npm": ">= 6.5.0"
```

Comment:
FYI These are above what we’ve required for lwc itself. 

What do sfdx and our VS Code extensions ship and require?

Reply:
8.9.4 is too low for the current CLI, the `sfdx-cli` plugin fails with that. As the CLI team recommends using LTS we're switching to that now, as 8.x is in maintenance mode.

The version change is not necessary for anything CLI related (yet), but it creates alignment.